### PR TITLE
Add SQL support content to Redshift connector

### PR DESCRIPTION
**Pull request by jhlodin**
Update the Redshift connector doc with a standardized SQL support section.

### CODE REVIEW

**Review by m57lyra — APPROVED** [review]
LGTM

**Review by rosewms — APPROVED** [review]

File: docs/src/main/sphinx/connector/redshift.rst [hunk 1]
```diff
@@ -77,13 +77,20 @@ Type mapping
...
+<sql-globally-available>` and :ref:`read operation <sql-read-operations>`
+statements, the connector supports the following features:
 
-* :doc:`/sql/grant`
-* :doc:`/sql/revoke`
-* :doc:`/sql/show-grants`
+* :doc:`/sql/insert`
+* :doc:`/sql/delete`
+* :doc:`/sql/show-roles`
+* :doc:`/sql/show-role-grants`
```
Comment (hashhar) [inline docs/src/main/sphinx/connector/redshift.rst:93]: I don't think these are supported.

**Review by hashhar — COMMENTED** [review]
See https://github.com/trinodb/trino/pull/8667#discussion_r677171808

File: docs/src/main/sphinx/connector/redshift.rst [hunk 2]
```diff
@@ -77,13 +77,18 @@ Type mapping
...
+Redshift. In addition to the :ref:`globally available
+<sql-globally-available>` and :ref:`read operation <sql-read-operations>`
+statements, the connector supports the following features:
 
-* :doc:`/sql/grant`
-* :doc:`/sql/revoke`
-* :doc:`/sql/show-grants`
+* :doc:`/sql/insert`
+* :doc:`/sql/delete`
+* :ref:`sql-schema-table-management`
```
Comment (hashhar) [inline docs/src/main/sphinx/connector/redshift.rst:92]: Turns out `sql-schema-table-management` isn't fine-grained enough for most connectors. Redshift and most JDBC connectors doesn't support ALTER SCHEMA but rest of it seems correct.

**Review by hashhar — APPROVED** [review]
LGTM % comment

File: docs/src/main/sphinx/connector/redshift.rst (hunk 2 above)
Comment (jhlodin) [reply docs/src/main/sphinx/connector/redshift.rst]: If that's the case, then the best solution would be to include the specific statements from schema table management that are supported. I added them all (except for ALTER SCHEMA) here, if you can take a look.

If it's the same for other JDBC connectors, we ought to do the same there.

Comment (hashhar) [reply docs/src/main/sphinx/connector/redshift.rst]: Thanks. Looks good now. See https://github.com/trinodb/trino/pull/8667#discussion_r677574922 for the state of ALTER SCHEMA. (Only Hive and Iceberg support it AFAIK).

Comment (jhlodin) [reply docs/src/main/sphinx/connector/redshift.rst]: Updated

**Review by hashhar — APPROVED** [review]

File: docs/src/main/sphinx/connector/redshift.rst [hunk 3]
```diff
@@ -77,13 +77,23 @@ Type mapping
...
 
-The following SQL statements are not yet supported:
-
-* :doc:`/sql/grant`
-* :doc:`/sql/revoke`
-* :doc:`/sql/show-grants`
+The connector provides read access and write access to data and metadata in
+Redshift. In addition to the :ref:`globally available
+<sql-globally-available>` and :ref:`read operation <sql-read-operations>`
+statements, the connector supports the following features:
```
Comment (hashhar) [inline docs/src/main/sphinx/connector/redshift.rst:88]: Should this be?
```suggestion
statements, the connector supports the following statements:
```

Comment (jhlodin) [reply docs/src/main/sphinx/connector/redshift.rst:88]: See https://github.com/trinodb/trino/pull/8668#discussion_r678538095, leaving as-is for now

**Review by hashhar — APPROVED** [review]

**Pull request merged**